ros-controls.jazzy.repos

```diff
@@ -34,4 +34,16 @@ repositories:
   ackermann_msgs:
     type: git
     url: https://github.com/ros-drivers/ackermann_msgs.git
-    version: ros2
\ No newline at end of file
+    version: ros2
+  sdformat_urdf:
+    type: git
+    url: https://github.com/ros/sdformat_urdf.git
+    version: jazzy
+  gazebo-release/sdformat_vendor:
+    type: git
+    url: https://github.com/gazebo-release/sdformat_vendor.git
+    version: jazzy
+  gazebo-release/gz_tools_vendor:
+    type: git
+    url: https://github.com/gazebo-release/gz_tools_vendor.git
+    version: jazzy
```

Comment:
Do we need to add these to the .repos?
Why?

Reply:
because they are not part of https://raw.githubusercontent.com/ros2/ros2/jazzy/ros2.repos